src/Elmish/React.js

```diff
@@ -36,5 +36,37 @@ exports.createElement_ = function(component, props, children) {
   //
   //  return React.createElement(component, props, ...children)
   //
-  return React.createElement.apply(null, [component, props].concat(children))
+  //  HTML `data-` attributes
```

Comment:
While I'm very much for having as much comments as possible, this is not what I meant when I asked for comments.

I meant to add comments to the PureScript code in relevant places, so that somebody reading docs on Pursuit can discover this feature.